pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -70,7 +70,7 @@ func (uc *UpgradeCluster) UpgradeCluster(subscriptionID uuid.UUID, resourceGroup
 	}
 
 	switch ucs.OrchestratorProfile.OrchestratorVersion {
```

Comment:
Synced offline

The upgrade code path is working out of the box. So it will not be the blocker here.
I will align upgrade code with this change later, as documented #1114. 

Thx for illustrating me how to use upgrade function. And seeing the upgrade function works is exciting.